File: ployglots/mta-ployglot-pkg-vault/packages/py/examples/sdk/05_get_secret_safe.py
from polyglot_vault_file import VaultFileSDK

sdk = VaultFileSDK.create().with_env_path("packages/py/tests/fixtures/.env.mixed").build()
sdk.load_config()
print("known:", sdk.get_secret_safe("SHARED").model_dump_json())
print("missing:", sdk.get_secret_safe("NOT_PRESENT").model_dump_json())

File: ployglots/mta-ployglot-pkg-vault/packages/ts/examples/sdk/05-get-secret-safe.mjs
import { VaultFileSDK } from '@polyglot/vault-file';

const sdk = VaultFileSDK.create()
  .withEnvPath('packages/ts/tests/fixtures/.env.mixed')
  .build();
sdk.loadConfig();
console.log('known:', JSON.stringify(sdk.getSecretSafe('SHARED')));
console.log('missing:', JSON.stringify(sdk.getSecretSafe('NOT_PRESENT')));
